sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Processor/EventProcessorClientTests.cs

```diff
@@ -951,8 +951,6 @@ public async Task ProcessorRaisesProcessEventHandlerWhenEventsAreRead()
                 Assert.That(capturedEventArgs[index].CancellationToken, Is.EqualTo(cancellationSource.Token), $"The cancellation token should have been propagated at index { index }.");
                 Assert.That(async () => await capturedEventArgs[index].UpdateCheckpointAsync(), Throws.Nothing, $"A checkpoint should be allowed for the event at index { index }.");
 
-                var expectedStart = EventPosition.FromOffset(capturedEventArgs[index].Data.Offset);
```

Comment:
Uh...  was this not actually being used?   😞

Reply:
Yeah I was looking at all the tests with FromOffset to see which needed to be switched to SequenceNumber and couldn't find where this was being used.

Reply:
I shall go feel bad about myself now.